src/docs/src/whatsnew/3.3.rst

```diff
@@ -83,6 +83,7 @@ Features and Enhancements
 * :ghissue:`4347`: Remove failed ``couch_plugins`` experiment, fixes more
   ``undef`` errors.
 * :ghissue:`4348`: Fix ``undef`` error in ``weatherreport``.
+* :ghissue:`4353`: Allow starting of more than one replication job. (D’OH!)
```

Comment:
"D'OH"?  Should not the release notes stick to a more formal technical style?  I am just curious.

Reply:
Nah, we like to be a bit informal and whimsical with our release notes. They often reflect the personality of whoever writes them.  We have [Star Trek references](https://docs.couchdb.org/en/stable/whatsnew/3.3.html#highlights), [Rainbow Gnomes](https://docs.couchdb.org/en/stable/whatsnew/3.1.html#features-and-enhancements), [Party cats](https://docs.couchdb.org/en/stable/whatsnew/3.0.html#upgrade-notes) and other such things.

Reply:
Thanks for the explanation :-)